homeworks/roman.strokan_Strokan/homework_3/z2.js

```diff
@@ -0,0 +1,20 @@
+let a = 1;
+let b;
+
+function exp2(n) {
+    if (a > n) {
+        return false;
+    }
+    if (a === n) {
+        return true;
+    }
+    a *= 2;
+    return exp2(n);
+}
+
```

Comment:
you function has side effects mutating variable out of it's scope, this is also a bad practice in general, avoid this